config/locales/en.txt

```diff
@@ -1477,9 +1477,32 @@
   name_map_no_maps: No observations of [name] have mappable location information.
 
   # observer/news
-  news_title: "Latest Release: August 3, 2015"
-  news_header: This page provides notes on the latest release of the [:app_title] website.
-  news_content: "December 13, 2015. <u>Rails 4.2.5</u>\n\nAt long last the Mushroom Observer website is up to date with the latest stable version of Ruby on Rails (4.2.5). This has not been true since at least March of 2009.  The upgrade process started last August when we were still using the 2.1.1 version of Ruby on Rails.\n\nThis has been a huge team effort by all the volunteer developers on the project. Thanks to every one of you for helping to make this happen.\n\nFor those who are interested in the technical low-down of this specific release, we got to 4.1 last month and the biggest change between 4.1 and 4.2 was a new HTML parsing library that required many minor code changes throughout out code base. There have also been some changes to how email is handled and a variety of other small changes to the code base.  You should see no new changes in the site due to this latest upgrade, but of course <a href=\"/observer/ask_webmaster_question\">send us a note</a> if you see anything wrong."
+  news_title: "Latest Release: August 16, 2015"
```

Comment:
1.  Thanks so much for taking the time to deal with the pending PRs.
2.  Yes, I intended 2016.  Good catch!  I'll fix it shortly.
3.
​GitHub's ​w
anting you to fork my repo
​results from
 my checking the "Allow edits from maintainers" checkbox when I made the
PR. GitHub
​thinks this
 make
​s​
things easier for maintainers:

"If you've ever wanted to make minor changes before merging a pull request,
now you can. When a user opens a pull request from a fork, they'll be given
a new option that allows the upstream repository contributors to
collaborate with them on their new branch.
"Simply clone the contributor's fork and, if you've been granted
permission, you'll be able to push changes to the head branch on the open
pull request, even if you don't normally have permission to push to the
fork."

B
​ut if it complicates your workflow, (a) I won't check that box, and (b)
I'll put in a PR to remove the instructions I just added to
developer-workflow.md (telling developers to check the box).

​
On Fri, Sep 9, 2016 at 10:48 AM Nathan Wilson <notifications@github.com>
wrote:

In config/locales/en.txt
> <https://github.com/MushroomObserver/mushroom-observer/pull/239#discussion_r78220729>
> :
>
> > @@ -1477,9 +1477,32 @@
> >    name_map_no_maps: No observations of [name] have mappable location information.
> >
> >    # observer/news
> > -  news_title: "Latest Release: August 3, 2015"
> > -  news_header: This page provides notes on the latest release of the [:app_title] website.
> > -  news_content: "December 13, 2015. <u>Rails 4.2.5</u>\n\nAt long last the Mushroom Observer website is up to date with the latest stable version of Ruby on Rails (4.2.5). This has not been true since at least March of 2009.  The upgrade process started last August when we were still using the 2.1.1 version of Ruby on Rails.\n\nThis has been a huge team effort by all the volunteer developers on the project. Thanks to every one of you for helping to make this happen.\n\nFor those who are interested in the technical low-down of this specific release, we got to 4.1 last month and the biggest change between 4.1 and 4.2 was a new HTML parsing library that required many minor code changes throughout out code base. There have also been some changes to how email is handled and a variety of other small changes to the code base.  You should see no new changes in the site due to this latest upgrade, but of course <a href=\"/observer/ask_webmaster_question\">send us a note</a> if you see anything wrong."
> > +  news_title: "Latest Release: August 16, 2015"
>
> I think you intend this to be 2016. I tried doing a quick edit of it, but
> it wanted me to fork your repo. Hopefully this is an easy fix you can get
> it quick.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/MushroomObserver/mushroom-observer/pull/239/files/1ecb5dfa543152bad774684f182e74d8dc6093f0#r78220729>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACxlFGsSzVT9sKj28HZAMf_1FmlyWGDks5qoZvigaJpZM4J2bQ_>
> .
>